lld/ELF/BPSectionOrderer.h

```diff
@@ -102,16 +102,41 @@ class BPSectionELF : public BPSectionBase {
                             &sectionToIdx) const override {
     constexpr unsigned windowSize = 4;
 
-    // Calculate content hashes
     size_t size = isec->content().size();
     for (size_t i = 0; i < size; i++) {
       auto window = isec->content().drop_front(i).take_front(windowSize);
       hashes.push_back(xxHash64(window));
     }
 
-    // TODO: Calculate relocation hashes.
-    // Since in ELF, relocations are complex, but the effect without them are
-    // good enough, we just use 0 as their hash.
+    for (const auto &r : isec->relocations) {
```

Comment:
@ellishg @MaskRay Hi, I've updated the relocations hashes by the `isec->relocations` which seems an common interface to access relocation. Could you please review these?

Reply:
Nice, it seems good. Did you see an uncompressed or compressed size win with this change?